docs/src/main/asciidoc/spanner.adoc

```diff
@@ -665,10 +665,13 @@ Queries:
 [source,java]
 ----
 // a query with options:
-SpannerQueryOptions spannerQueryOptions = new SpannerQueryOptions().setTimestamp(Timestamp.now());
+SpannerQueryOptions spannerQueryOptions = new SpannerQueryOptions().setTimestamp(myTimestamp);
 List<Trade> trades = this.spannerTemplate.query(Trade.class, Statement.of("SELECT * FROM trades"), spannerQueryOptions);
 ----
 
+You can also read with https://cloud.google.com/spanner/docs/timestamp-bounds[*bounded staleness*] by setting `.setTimestampBound(TimestampBound.ofMinReadTimestamp(myTimestamp))` on the query and read options objects.
```

Comment:
would be good to have code snippets to show different staleness modes

Reply:
There are literally just those two. The snippet above shows `.setTimestamp` and this one here inline shows bounded staleness.

Reply:
@dmitry-s